tests/i18n/tests.py

```diff
@@ -2175,8 +2175,17 @@ def test_other_lang_with_prefix(self):
         response = self.client.get("/fr/simple/")
         self.assertEqual(response.content, b"Oui")
 
-    def test_unprefixed_language_other_than_accept_language(self):
+    def test_unprefixed_language_with_accept_language(self):
+        """
+        When user provides an "Accept-Language" or provides a request
+        language in cookies, we should honour that and redirect to the
+        appropriate language if exists
```

Comment:
I can't see it explicitly mentioned in the coding standard but conventionally all sentences in comments, docstrings & commit messages end with a full stop 😁

Reply:
Thanks. I also noticed that the docstring says more than what it actually tests (it only tests the Accept Language, not the Cookies)

I will make a test to split the cookie test and the Accept language test, although that should be the responsibility of `get_language_from_request` unit tests.

Doesn't hurt to test the integration tho, because that was the issue described in the link in the description :)